managers/asset-operations-manager.js

```diff
@@ -333,8 +324,9 @@ class AssetOperationsManager {
             ));
 
         let tokenId;
-        if(paranetUAL == null) {
-            tokenId = await this.blockchainService.createAsset(
+        let receipt;
+        if (paranetUAL == null) {
+            [tokenId, receipt] = await this.blockchainService.createAsset(
```

Comment:
I would suggest returning object here, then destructuring would look like that:
```suggestion
            ({ tokenId, receipt } = await this.blockchainService.createAsset(
```